book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Please kindly update your add_rental method to properly achieve the many to many relationship
- code snippet
```ruby
 def add_rental(person, date)
    Rental.new(date, self, person)
  end
```